packages/react-components/src/components/video/index.tsx

```diff
@@ -39,6 +39,7 @@ type Props = {
     height?: number
     volume?: number
     className?: string
+    copied?: boolean
```

Comment:
@kylebcarlson this new property `copied` doesn't have any code associated with it. It looks like you're adding a link button in the overlay, which you can do in the web app, no? Maybe in one of the overlay components [here](https://github.com/Giphy/web-app/blob/master/app/static/js/shared/components/video-inline-play/index.tsx#L104)?

Reply:
made some changes to this PR https://github.com/Giphy/web-app/pull/3387/files
added the `copied` property so this component expects it: `shared/components/video-player/sdk-video-with-ga.tsx`

Reply:
@kylebcarlson I think we should find a way to add it to the web app then, follow the pattern of `gridPosition` there which looks to be used in a similar way and is not part of the SDK

Reply:
ah yeah, damn i should have seen that.